arangodb-net-standard/DatabaseApi/DatabaseApiClient.cs

```diff
@@ -6,9 +6,20 @@
 
 namespace ArangoDBNetStandard.DatabaseApi
 {
+    /// <summary>
+    /// A client for interacting with ArangoDB Databases endpoints,
+    /// implementing <see cref="IDatabaseApiClient"/>.
+    /// </summary>
     public class DatabaseApiClient : ApiClientBase, IDatabaseApiClient
     {
+        /// <summary>
+        /// The transport client used to communicate with the ArangoDB host.
+        /// </summary>
         private IApiClientTransport _client;
```

Comment:
Should this be `protected` now?

Reply:
Definitely, thanks for catching that. I will update the pull request.